### PR TITLE
Update microsoft-teams-rooms-premium-rbac.md

### PR DESCRIPTION
**Pull request by altsou**
Adding new section and details for new features added some time ago.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/microsoft-teams-rooms-premium-rbac.md [hunk 1]
```diff
@@ -69,9 +69,23 @@ The following table summarizes what each role can do.
 |Role management    |View         |&#10004;           |         ||
 |    |Modify         |&#10004;           |         ||
 
+## Create a custom role
+
+If the built-in roles do not suit your organizational needs, you can create a role and configure its permissions as desired. To create a role, you must be a Global Administrator or Managed Service Administrator. 
+
+1. In the left navigation of the Microsoft Teams Rooms managed service portal, go to **Settings** > **Roles**.
+2. Select **Create role**
+3. On the **General settings** page, under **Role properties** enter a name for this role. Under **Description**, enter details about this role. Choose **Next**
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium-rbac.md:78]:
```suggestion
2. Select **Create role**.
3. On the **General settings** page, under **Role properties** enter a name for this role. Under **Description**, enter details about this role. Choose **Next**.
```

File: Teams/rooms/microsoft-teams-rooms-premium-rbac.md [hunk 2]
```diff
@@ -69,9 +69,23 @@ The following table summarizes what each role can do.
...
+4. On the **Permissions** page, under **Role permissions** choose the permissions for this role by selecting the appropriate check-boxes . Choose **Next** to create the first assignment for this role.
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium-rbac.md:79]:
```suggestion
4. On the **Permissions** page, under **Role permissions** choose the permissions for this role by selecting the appropriate check-boxes. Choose **Next** to create the first assignment for this role.
```

File: Teams/rooms/microsoft-teams-rooms-premium-rbac.md [hunk 3]
```diff
@@ -69,9 +69,23 @@ The following table summarizes what each role can do.
...
+5. On the **Assignments** page, under **Assignment properties**, enter a name for this assignment. The description is optional. Under **Notification settings** select the **E-mail notifications** check-box if users of this role should recieve email notications from the service on rooms in the **Scope** of this assignment. Choose **Next**.
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium-rbac.md:80]:
```suggestion
5. On the **Assignments** page, under **Assignment properties**, enter a name for this assignment. The description is optional. Under **Notification settings** select the **E-mail notifications** check-box if users of this role should receive email notifications from the service on rooms in the **Scope** of this assignment. Choose **Next**.
```

File: Teams/rooms/microsoft-teams-rooms-premium-rbac.md [hunk 4]
```diff
@@ -69,9 +69,23 @@ The following table summarizes what each role can do.
...
+4. On the **Permissions** page, under **Role permissions** choose the permissions for this role by selecting the appropriate check-boxes . Choose **Next** to create the first assignment for this role.
+5. On the **Assignments** page, under **Assignment properties**, enter a name for this assignment. The description is optional. Under **Notification settings** select the **E-mail notifications** check-box if users of this role should recieve email notications from the service on rooms in the **Scope** of this assignment. Choose **Next**. 
+6. On the **Members** page, in the **Search for user or security group** box, enter the name of a user or security group in your tenant to which you want to give permissions, and then complete the selection. Choose **Next**. 
+7. On the **Scope** page, in the **Search for room or room group** box, type the name of either a room or room group that the user will be allowed to manage. Choose **Next**.
+8. On the **Finish** page, review the details of the role and assignment. If you're satisfied with the configuration, choose **Add new role**. If you want to edit a section, use the **Previous** button or select the step in the left navigation.  
+
+
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium-rbac.md:85]:
```suggestion
```

File: Teams/rooms/microsoft-teams-rooms-premium-rbac.md [hunk 5]
```diff
@@ -82,7 +96,7 @@ To assign roles, you must be a Global Administrator or Managed Service Administr
 
     :::image type="content" source="../media/microsoft-teams-rooms-premium-role-assignments.png" alt-text="Screenshot of Add option to add a role.":::
 
-4. On the **General settings** page, under **Assignment properties**, enter a name for this assignment. The description is optional. Choose **Next.**
+4. On the **General settings** page, under **Assignment properties**, enter a name for this assignment. The description is optional. Under **Notification settings** select the **E-mail notifications** check-box if users of this role should recieve email notications from the service on rooms in the **Scope** of this assignment. Choose **Next**.
```
Comment (JohanFreelancer9) [inline Teams/rooms/microsoft-teams-rooms-premium-rbac.md:99]:
```suggestion
4. On the **General settings** page, under **Assignment properties**, enter a name for this assignment. The description is optional. Under **Notification settings**, select the **E-mail notifications** check-box if users of this role should receive email notifications from the service on rooms in the **Scope** of this assignment. Choose **Next**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @altsou, kindly commit the suggested changes. Many thanks.

**Comment by altsou** [comment]
Changes look good to me. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by altsou — COMMENTED** [review]
All looks good to me. When can we publish?

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**